src/acktest/k8s/resource.py

```diff
@@ -113,8 +113,10 @@ def load_and_create_resource(resource_directory: Path,
 
 def _get_k8s_api_client() -> ApiClient:
     global _k8s_api_client
-    if _k8s_api_client is None:
-        _k8s_api_client = config.new_client_from_config()
+    # Create new client everytime to avoid token refresh issues
+    # https://github.com/kubernetes-client/python/issues/741
+    # https://github.com/kubernetes-client/python-base/issues/125
+    _k8s_api_client = config.new_client_from_config()
     return _k8s_api_client
```

Comment:
Can't we remove the global _k8s_api_client and make the entire function just:

```
return config.new_client_from_config()
```
?

Reply:
yep, just realized the same :)